release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -44,6 +44,19 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-7-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-7-aws-c2s-region"]
+==== Installing a cluster into an AWS C2S region
+
+You can now install a cluster on Amazon Web Services (AWS) into a Commercial Cloud Services (C2S) region. Because C2S regions do not have {op-system} AMIs published by Red Hat, you must upload a custom AMI that belongs to that region. You are also required to include the CA certificates for C2S in the `additionalTrustBundle` field of the `install-config.yaml` file during cluster installation. The installation program does not support destroying a cluster deployed to a C2S region; you must manually remove the resources of the cluster.
```

Comment:
I wonder if Commercial Cloud Services is the name that we should be using. Amazon calls it the Secret Region.

Reply:
@staebler From what I gathered, the acronym C2S stands for *Commercial Cloud Services*. With that said (and as you mentioned), I've seen Amazon's docs refer to this as the AWS Secret Region built as part of the Commercial Cloud Services (C2S) contract, so I agree this is a bit weird keeping it the way I referred to it here.

I guess the biggest question here is how much initial background do we want to provide on what C2S is. We typically want to spell out acronyms when they're first mentioned, but if it doesn't make sense in this case, we can make an exception.

I've seen third party docs (e.g., [here](https://docs.gigamon.com/doclib59/Content/GV-Cloud-AWS-SecretRegions/AWS_Secret_Regions_Overview.html)) refer to it as the Commercial Cloud Services (C2S) Secret Region, so perhaps taking the middle ground is the best way (leaving off the drawn out *Commercial Cloud Services*, while also adding *Secret Region*).

Do you think referring to it as `AWS C2S Secret Region` here (and throughout docs) is the most appropriate way forward?